src/qibolab/_core/instruments/emulator/operators.py

```diff
@@ -1,23 +1,39 @@
-from qutip import basis, sigmap, sigmax, sigmaz, tensor
+"""Useful operators in qutip for n levels transmon."""
 
-N_LEVELS = 2
-"""Levels for transmon system."""
+from qutip import Qobj, basis, create, destroy, tensor
 
-STATE_0 = basis(N_LEVELS, 0)
-"""State 0."""
-STATE_1 = basis(N_LEVELS, 1)
-"""State 1."""
 
-INITIAL_STATE = tensor(STATE_0)
-"""System initial state."""
+def transmon_create(n: int) -> Qobj:
+    """Creation operator for n levels system."""
+    return create(n)
 
-SIGMAZ = sigmaz()
-"""Qubit destruction operator."""
-QUBIT_DRIVE = sigmax()
-"""Qubit drive term."""
 
-# TODO: check these operators
-L1 = sigmap()
-"""Time relaxation operator."""
-L2 = sigmaz()
-"""Dephasing operator."""
+def transmon_destroy(n: int) -> Qobj:
+    """Destruction operator for n levels system."""
+    return destroy(n)
```

Comment:
Do you believe it is actually worth to keep these functions? They are mainly typed aliases